src/main/java/core/basesyntax/service/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.db.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity")

```

Comment:
Move header and separator to the constant fields. Also, better use System.linesepartor()